### PR TITLE
tappc190 removing unecessary parameters 

### PR DESCRIPTION
**Pull request by leclaist**
[*tappc190*](https://jira.eng.vmware.com/browse/TAPPC-190)

* Removed parameter sample app name 
* Removed bastion os
* Removed bastion ssh port
* Added eks cluster name default value 
* Added qss3 bucket prefix default value 
* removed commented out content in post deploy doc

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/deployment_guide/partner_editable/post_deployment.adoc [hunk 1]
```diff
@@ -145,14 +129,9 @@
 
 |`EKSClusterName`
```
Comment (satya-dillikar) [inline docs/deployment_guide/partner_editable/post_deployment.adoc]: EKSClusterName can be totally removed. Instead, we can have an auto-generated name in the code

File: docs/deployment_guide/partner_editable/post_deployment.adoc [hunk 2]
```diff
@@ -174,22 +153,6 @@
 |Maximum number of available nodes for each {partner-product-acronym} EKS cluster in auto scaling.
 |====
 
-[%autowidth.stretch]
```
Comment (satya-dillikar) [inline docs/deployment_guide/partner_editable/post_deployment.adoc:177]: I am not sure if we want to remove the SSH config.

**Review by satya-dillikar — CHANGES_REQUESTED** [review]
Please also add the actual CFT template files code changes in the other PR or a reference in this PR.

File: docs/deployment_guide/partner_editable/post_deployment.adoc (hunk 1 above)
Comment (leclaist) [reply docs/deployment_guide/partner_editable/post_deployment.adoc]: removed

**Review by satya-dillikar — APPROVED** [review]

**Review by satya-dillikar — COMMENTED** [review]
updated branch

**Pull request merged**